### PR TITLE
Refactor _actions/_top.py into two files

### PR DESCRIPTION
**Pull request by lleshchi**
Closes: #738

### CODE REVIEW

File: src/stratis_cli/_actions/_pool.py [hunk 1]
```diff
@@ -0,0 +1,664 @@
...
+    :param proxy: proxy to the top object in stratisd
+    :return: list of pool UUIDs as strings
+    :rtype: list of str
+    :raises StratisCliPropertyNotFoundError:
+    :raises StratisCliEnginePropertyError:
+    """
+    return _fetch_property(proxy, "LockedPools")
+
+
+def _fetch_property(proxy, property_name):
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_pool.py:155]: I _think_ this is the only method that's shared between the two files. Best place for it is ```_action/_utils.py```. There's another function in ```_utils.py``` called ```fetch_property``` already, you can rename that one to something more appropriate, like maybe ```unpack_property```.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just one request...seems like a perfectly reasonable division to me.

File: src/stratis_cli/_actions/_pool.py (hunk 1 above)
Comment (lleshchi) [reply src/stratis_cli/_actions/_pool.py:155]: @mulkieran moving `fetch_property` into `_action/_utils.py` creates a circular import. `_action/_data.py` imports `get_timeout` from `_action/_utils.py`, and `_action/_utils.py` imports `FetchProperties` from `_action/_data.py`. 

How do you feel about moving `get_timeout` to `_action/_data.py`, as that is the only place outside of testing where it is used?

Comment (mulkieran) [reply src/stratis_cli/_actions/_pool.py:155]: > @mulkieran moving `fetch_property` into `_action/_utils.py` creates a circular import. `_action/_data.py` imports `get_timeout` from `_action/_utils.py`, and `_action/_utils.py` imports `FetchProperties` from `_action/_data.py`.
> 
> How do you feel about moving `get_timeout` to `_action/_data.py`, as that is the only place outside of testing where it is used?

That seems like a good solution.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Looks ready. Please close this and open a new PR for final review.

**Pull request closed without merging**